src/client/css/style.scss

```diff
@@ -0,0 +1,88 @@
+$btnColor: #4A90E2;
+$formColor: #FFFFFF;
+$bgColor: rgb(246,249,255);
+$defaultColor: #9B9B9B;
+$titleFont: Helvetica-Bold;
+
+* {
+  margin: 0;
```

Comment:
i will create a new story for the basics settings, you can remove this reset css